src/errors/article-has-no-title.ts

```diff
@@ -0,0 +1,7 @@
+import { schema } from 'rdf-namespaces';
```

Comment:
👍 (should do it elsewhere too)

Reply:
I have deliberately avoided using it in tests. Thoughts?